src/gmusic-theme.js

```diff
@@ -123,8 +165,8 @@ window.GMusicTheme = class GMusicTheme {
   }
 
   _refreshStyleSheet() {
-    // DEV: Take the current style string and put it in the style element in the DOM
-    this.styleElement.innerHTML = this._substituteColors(BASE_CSS);
+    // DEV: Take the current style string and put it in the style element in the DOM)
+    this.styleElement.innerHTML = this._substituteColors((this.enabled === GMusicTheme.STATES.ENABLED_ALL ? BASE_CSS + BASE_HIGHLIGHT_CSS : BASE_HIGHLIGHT_CSS));
```

Comment:
This line has grown pretty long. Maybe the ternary would be best suited as another line of code:

```
var cssContent = this.COMMON_CSS;
if (this.enabled === GMusicTheme.STATES.ENABLED_ALL) {
  cssContent += BASE_HIGHLIGHT_CSS;
}
```